canonical-web-and-design/manager.assets.ubuntu.com
# Use error handlers from flask-base

Use the standard error handlers provided by flask base with minimal templates.

## QA

- In the demo go to `/no` and see a page rendered saying `404`
- Try to make a search and see a page rendered saying `500` (will happen because no token is set)
